apps/quickstart-integrations-monday-code/index.js

```diff
@@ -0,0 +1,104 @@
+import express from "express";
+import { Logger } from "@mondaycom/apps-sdk";
+import { transformText } from "./src/transformation-service.js";
+import { authorizeRequest } from "./src/middleware.js";
+import { changeColumnValue, getColumnValue } from "./src/monday-api-service.js";
+import { getSecret, isDevelopmentEnv, getEnv } from "./src/helpers.js";
+import dotenv from "dotenv";
+dotenv.config();
+
+const logTag = "ExpressServer";
+const PORT = "PORT";
+const SERVICE_TAG_URL = "SERVICE_TAG_URL";
+const TO_UPPER_CASE = "TO_UPPER_CASE";
+const TO_LOWER_CASE = "TO_LOWER_CASE";
+
+const logger = new Logger(logTag);
+const currentPort = getSecret(PORT); // Port must be 8080 to work with monday code
+const currentUrl = getSecret(SERVICE_TAG_URL);
+
+const app = express();
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.status(200).send({ message: "healthy" });
+});
+
+app.post("/monday/execute_action", authorizeRequest, async (req, res) => {
+  logger.info(
```

Comment:
consider using the Logger from the SDK, as we have an issue with supporting this with fetching from CLI, see this https://github.com/mondaycom/apps-sdk#logger